classes/class-ui-panel.php

```diff
@@ -257,7 +257,17 @@ static public function uabb_templates_data( $templates, $status = 'exclude' ) {
 	 */
 	function builder_ui_bar_buttons( $buttons ) {
 
-		$simple_ui    = ! FLBuilderModel::current_user_has_editing_capability();
+		if( defined( 'FL_BUILDER_VERSION' ) ) {
+			$p = '#(\.0+)+($|-)#';
+			$ver1 = preg_replace($p, '', FL_BUILDER_VERSION);
+		    $ver2 = preg_replace($p, '', '1.10');
+			if( version_compare( $ver1, $ver2 ) < 0 ) {
```

Comment:
`preg_match`, `preg_replace` and `version_compare` seem a overkill here, can we use something simple like 
```
if ( is_callable( 'FLBuilderUserAccess::current_user_can' ) ) {
	// use FLBuilderUserAccess::current_user_can()
} else {
	// use FLBuilderModel::current_user_has_editing_capability()
}
```